src/Sylius/Bundle/ApiBundle/test/config/packages/test_with_swiftmailer/filesystem.yaml

```diff
@@ -0,0 +1,4 @@
+services:
+    test.filesystem.public:
```

Comment:
Probably it would be better to make filesystem public in compiler pass or something, so I see it as a quick-fix (needed only to test swiftmailer) 🚀